src/common/aes/aes.h

```diff
@@ -11,10 +11,7 @@
 #ifndef OQS_AES_H
 #define OQS_AES_H
 
-#include <stdint.h>
-#include <stdlib.h>
```

Comment:
This is a very minor style point, but I would prefer to keep the `stdint.h` and `stdlib.h` (or `stddef.h`) includes and not rely on them being transitively included via `aes_ops.h`. The same applies for the SHA2 and SHA3 header files.

Reply:
Sure, added them back.